### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by tireddy2**

### CODE REVIEW

File: draft-ietf-add-resolver-info.md [hunk 1]
```diff
@@ -117,9 +117,11 @@ Reputation:
    using the RESINFO RR type and QNAME of "resolver.arpa". In this case, a client has to contend
    with the risk that a resolver does not support RESINFO. The resolver might
    pass the query upstream, and then the client can receive a positive RESINFO response either
-   from a legitimate upstream DNS resolver or an attacker. If a client sees the RESINFO in the
-   Answer section, it can detect that the response is not provided by the resolver
-   and discards the response.
+   from a legitimate upstream DNS resolver or an attacker. The client in the DNS query MUST
```
Comment (boucadair) [inline draft-ietf-add-resolver-info.md]:
```suggestion
   from a legitimate upstream DNS resolver or an attacker. The DNS client MUST
```

File: draft-ietf-add-resolver-info.md [hunk 2]
```diff
@@ -117,9 +117,11 @@ Reputation:
...
+   set the Recursion Desired (RD) bit set to 0 to ensure the response is provided by the resolver.
```
Comment (boucadair) [inline draft-ietf-add-resolver-info.md]:
```suggestion
   set the Recursion Desired (RD) bit of the query to 0 to ensure that the response is provided by the resolver.
```

File: draft-ietf-add-resolver-info.md [hunk 3]
```diff
@@ -117,9 +117,11 @@ Reputation:
...
+   If the resolver does not support RESINFO, it will return an authoritative name error.
+   In addition, if a client sees the RESINFO in the Answer section, it can detect that
```
Comment (boucadair) [inline draft-ietf-add-resolver-info.md:123]: I think we should drop this last sentence

**Review by boucadair — CHANGES_REQUESTED** [review]

File: draft-ietf-add-resolver-info.md (hunk 3 above)
Comment (tireddy2) [reply draft-ietf-add-resolver-info.md:123]: Yes

**Pull request merged**